### DISCUSSION TITLE
Web Application: Forbidden, Referrer Invalid - Question about Headers

### DISCUSSION THREAD

**Discussion by jack-bobs** [gravitational/teleport#14229] [Q&A] [answered]
Hi, 

So I currently have an application that is connected to our cluster. When I try to log through Teleport to access our application, I will get the following screenshot:

![image](https://user-images.githubusercontent.com/108336827/178017191-cbd54624-74a4-4007-b783-b41fc96f089f.png)
(Forbidden. Referrer Invalid) 

Would I need to add headers into my teleport.yaml file on the container which is hosting an application?

**Comment by webvictim** [comment, marked as answer]
That looks like a limitation imposed by whatever application you're proxying. Theoretically you can set a referer header:

```yaml
rewrite:
  headers:
  - "Referer: expected.referer.com"
```

See the docs here: https://goteleport.com/docs/application-access/guides/connecting-apps/#headers-passthrough
